postgres/answers/12_explanation.md

```
@@ -0,0 +1,21 @@
+       price
+  FROM products
+ WHERE price > (
+       SELECT AVG(price)
+         FROM products
+ );
+```
+
+## 解説
+
+- サブクエリ`(SELECT AVG(price) FROM products)`で全商品の平均価格を取得しています。
+- `WHERE price > (...)`で平均価格より高い商品を抽出しています。
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character
null

(MD047, single-trailing-newline)